src/DataWrangling/restoring.jl

```diff
@@ -19,50 +20,50 @@ import Oceananigans.OutputReaders: new_backend, update_field_time_series!
 @inline instantiate(T::DataType) = T()
 @inline instantiate(T) = T
 
-struct ECCONetCDFBackend{N, C, I, M} <: AbstractInMemoryBackend{Int}
+struct NetCDFBackend{N, C, I, M} <: AbstractInMemoryBackend{Int}
     start :: Int
     length :: Int
     inpainting :: I
     metadata :: M
 
-    function ECCONetCDFBackend{N, C}(start::Int, length::Int, inpainting, metadata) where {N, C}
+    function NetCDFBackend{N, C}(start::Int, length::Int, inpainting, metadata) where {N, C}
         M = typeof(metadata)
         I = typeof(inpainting)
         return new{N, C, I, M}(start, length, inpainting, metadata)
     end
 end
 
-Adapt.adapt_structure(to, b::ECCONetCDFBackend{N, C}) where {N, C} = ECCONetCDFBackend{N, C}(b.start, b.length, nothing, nothing)
+Adapt.adapt_structure(to, b::NetCDFBackend{N, C}) where {N, C} = NetCDFBackend{N, C}(b.start, b.length, nothing, nothing)
 
 """
-    ECCONetCDFBackend(length, metadata;
+    NetCDFBackend(length, metadata;
                       on_native_grid = false,
                       cache_inpainted_data = false,
                       inpainting = NearestNeighborInpainting(Inf))
 
-Represent an ECCO FieldTimeSeries backed by ECCO native netCDF files.
+Represent an observational/reanalysis FieldTimeSeries backed by native netCDF files.
 Each time instance is stored in an individual file.
 """
-function ECCONetCDFBackend(length, metadata;
+function NetCDFBackend(length, metadata;
                            on_native_grid = false,
                            cache_inpainted_data = false,
                            inpainting = NearestNeighborInpainting(Inf))
 
-    return ECCONetCDFBackend{on_native_grid, cache_inpainted_data}(1, length, inpainting, metadata)
+    return NetCDFBackend{on_native_grid, cache_inpainted_data}(1, length, inpainting, metadata)
 end
 
-Base.length(backend::ECCONetCDFBackend)  = backend.length
-Base.summary(backend::ECCONetCDFBackend) = string("ECCONetCDFBackend(", backend.start, ", ", backend.length, ")")
+Base.length(backend::NetCDFBackend)  = backend.length
+Base.summary(backend::NetCDFBackend) = string("NetCDFBackend(", backend.start, ", ", backend.length, ")")
 
-const ECCOFieldTimeSeries{N} = FlavorOfFTS{<:Any, <:Any, <:Any, <:Any, <:ECCONetCDFBackend{N}} where N
+const ObsFieldTimeSeries{N} = FlavorOfFTS{<:Any, <:Any, <:Any, <:Any, <:NetCDFBackend{N}} where N
```

Comment:
why `Obs` in the name here? The name implies that it can be only observations? But it can be anything really, right?

Reply:
Good point! I'll import the general function here

Reply:
Yes, I think we just want to extend the `FieldTimeSeries` method to work with Metadata.

Reply:
Latest commit extends this (https://github.com/CliMA/ClimaOcean.jl/pull/480/commits/94f22ec21d3c60085aa6d983bd0566dd5d2bf768)